### PR TITLE
Added pypenv config

### PR DESCRIPTION
**Pull request by PolaDinitz**
#12

### CODE REVIEW

File: vscode.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import json
-import requests
 import urllib.request
 
+import requests
+
```
Comment (syncush) [inline vscode.py:5]: why did you change the order of imports?

File: Pipfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+pylint = "*"
+
+[packages]
+requests = "*"
```
Comment (syncush) [inline Pipfile:10]: shouldn't we stick to a specific major version? just like npm, for example, `~3.1.2`

File: vscode.py (hunk 1 above)
Comment (PolaDinitz) [reply vscode.py:5]: conventions. first core packages, then outer and then inner project's packages

File: Pipfile (hunk 2 above)
Comment (PolaDinitz) [reply Pipfile:10]: this specific one is the latest... Nir used it so he's supposed to know the version he prefers.

Comment (syncush) [reply Pipfile:10]: so update the `pipenv file` to the latest version and let's not bump up in majors

File: vscode.py (hunk 1 above)
Comment (syncush) [reply vscode.py:5]: cool, resolved. @Guinness1759 @nirnir93 we need `pep8` or `autopep` in our pipeline for that kind of cases

File: Pipfile (hunk 2 above)
Comment (PolaDinitz) [reply Pipfile:10]: in this package the version doesn't matter. and it is in fact the latest version. It's stated in the lockfile so there will be no problems with the version of the package.

**Review by syncush — APPROVED** [review]

**Comment by syncush** [comment]
@nirnir93 @Guinness1759 can you please remove `codacy` ? we have to choose `CodeFactor` or `codacy`

**Comment by nirolevy** [comment]
> @nirnir93 @Guinness1759 can you please remove `codacy` ? we have to choose `CodeFactor` or `codacy`

@syncush 
1.  You have the same level of access as me, if you think we have to choose one then do it
2.  Why do we have to choose? aren't two better than one?

**Comment by syncush** [comment]
> > @nirnir93 @Guinness1759 can you please remove `codacy` ? we have to choose `CodeFactor` or `codacy`
> 
> @syncush
> 
> 1. You have the same level of access as me, if you think we have to choose one then do it
> 2. Why do we have to choose? aren't two better than one?

1. cool

2. what if there are 2 conflicting lint rules?  I would prefer to stick to one
